src/test/java/org/molgenis/vcf/report/AppIT.java

```diff
@@ -29,9 +36,29 @@ void test() throws IOException {
     String expectedReport =
         Files.readString(expectedReportPath)
             .replace("{{ inputPath }}", inputFile.replace("\\", "\\\\"))
+            .replace("{{ pedPaths }}", pedFiles.replace("\\", "\\\\"))
             .replace("{{ outputPath }}", outputFile.replace("\\", "\\\\"))
             .replace("{{ templatePath }}", templateFile.replace("\\", "\\\\"));
 
-    assertEquals(expectedReport, report);
+    //check the report api value with JSONAssert
```

Comment:
Is it possible to use ```assertAll``` such that all assertions are checked even if the first one fails?